src/components/home/balance-card.tsx

```diff
@@ -1,91 +1,73 @@
   const [loading, setLoading] = useState(false);
-  const [personTotalTime, setPersonTotalTime] = useAtom(personTotalTimeAtom);
   const adminMode = UserRoleUtils.adminMode();
+  const [usersFlextime, setUsersFlextime] = useState<Flextime>();
   const yesterday = DateTime.now().minus({ days: 1 });
+  const { flexTimeApi } = useLambdasApi();
 
   /**
-   * Initialize logged in person's time data.
+   * Initialize logged in users's time data.
    */
-  const getPersons = async () => {
+  const getUsersFlextimes = async () => {
     setLoading(true);
-    const loggedInPerson = persons.find(
-      (person: Person) =>
-        person.id === config.person.forecastUserIdOverride || person.keycloakId === userProfile?.id
-    );
-    if (loggedInPerson) {
+    const loggedInUser = user.find((user: User) => user.id === userProfile?.id);
+    if (loggedInUser) {
+      const severaUserId = getSeveraUserId(loggedInUser);
       try {
-        const fetchedPerson = await personsApi.listPersonTotalTime({
-          personId: loggedInPerson?.id,
-          timespan: Timespan.ALL_TIME,
-          before: yesterday.toJSDate()
+        const fetchedUsersFlextime = await flexTimeApi.getFlextimeBySeveraUserId({
+          severaUserId
         });
-        setPersonTotalTime(fetchedPerson[0]);
+        setUsersFlextime(fetchedUsersFlextime);
       } catch (error) {
-        setError(`${strings.error.fetchFailedGeneral}, ${error}`);
+        setError(`${strings.error.fetchFailedFlextime}, ${error}`);
       }
     }
     setLoading(false);
   };
 
   /**
-   * Get person total time if it is undefined or set to "all time"
+   * Fetch user's flextime data when users or userProfile changes.
    */
   useEffect(() => {
-    if (!personTotalTime) {
-      getPersons();
+    if (!usersFlextime) {
+      getUsersFlextimes();
     }
-  }, [persons]);
+  }, [user]);
 
   /**
-   * Renders person's total time
-   *
-   * @param personTotalTime PersonTotalTime
+   * Render user's flextime data.
    */
-  const renderPersonTotalTime = (personTotalTime: PersonTotalTime | undefined) => {
-    const balanceColor =
-      personTotalTime && personTotalTime.balance > 0
-        ? theme.palette.success.main
-        : theme.palette.error.main;
-
-    if (adminMode) {
-      return <Typography>{strings.placeHolder.notYetImplemented}</Typography>;
-    }
-    if (!personTotalTime && !loading && persons.length) {
-      return (
-        <Typography color={balanceColor}>{strings.error.fetchFailedNoEntriesGeneral}</Typography>
-      );
-    }
-    if (personTotalTime) {
-      return (
-        <Typography color={balanceColor}>{getHoursAndMinutes(personTotalTime.balance)}</Typography>
-      );
-    }
-    return <Skeleton />;
+  const renderUserFlextime = () => {
+    return (
+      <Typography variant="body1">
+        {strings.formatString(
+          strings.balanceCard.totalFlextimeBalance,
+          usersFlextime?.totalFlextimeBalance ?? ""
```

Comment:
If there is an error with fetching flex time an appropriate string should be renedered here rather than the current "Total flextime balance: hours"